src/qt/treectrl.cpp

```diff
@@ -432,6 +435,14 @@ class wxQTreeWidget : public wxQtEventSignalHandler<QTreeWidget, wxTreeCtrl>
         EmitEvent(expandedEvent);
     }
 
+    void OnTreeScrolled(int)
+    {
+        if (GetEditControl() == NULL)
+            return;
```

Comment:
Why not have the positive test (without the comparison to `NULL`) and avoid explicit `return`?

Please also mind the code style - in this case the spaces in `if ( condition )` - even if they look a bit awkward sometimes.

Reply:
This is just my personal preference to choose early out returns over further indentation. I will reverse it if you'd prefer, otherwise I'll just tidy up the include and if spacing.

Reply:
> This is just my personal preference to choose early out returns over further indentation.

Well, the rest of the function is only one more line long... So you chose indenting an extra  `return` line over indenting a single line.

One other guideline that I remember reading was arguing in favor of using a positive test [first], at least where possible and/or makes the code shorter.

But of course if you have a strong preference about this then it shouldn't really matter.

Reply:
Yes, I suppose that's fair in this situation, I may have written it like that pre-emptively in case there were more conditions/code needed to cancel the edit.